src/pages/dataTableUtilities/actions.js

```diff
@@ -60,31 +61,75 @@ export const closeBasicModal = () => ({
   type: 'closeBasicModal',
 });
 
-export const addMasterListItems = objectType => ({
-  type: 'addMasterListItems',
-  objectType,
-});
-
-export const addItem = (item, addedItemType) => ({
-  type: 'addItem',
-  item,
-  addedItemType,
-});
-
-export const editTheirRef = (value, pageObjectType) => ({
-  type: 'editTheirRef',
-  pageObjectType,
-  value,
-});
-
-export const editComment = (value, pageObjectType) => ({
-  type: 'editComment',
-  pageObjectType,
-  value,
-});
-
-export const deleteItemsById = (pageObjectType, object) => ({
-  type: 'deleteItemsById',
-  pageObjectType,
-  object,
+export const addMasterListItems = objectType => (dispatch, state) => {
```

Comment:
At this point is it worth while js docing the actions too?

Reply:
Yeah definitely - I think the file structure starts to matter more also (`thunk.js`/`action.js`? or `module1Actions.js` etc? Dunno!) .. I think all of these actions/reducers need to be tidied when we have a solid understanding of the groupings etc?

Reply:
Yea, let it mature a bit first 👍